mock_service/lib/dsl.py

```diff
@@ -109,8 +120,13 @@ def __resolve_value(action_context: ActionContext, value: Any) -> Any:
                     f"Unsupported literal: {value!r} in non event-triggered behavior!"
                 )
         elif value.startswith("$"):
-            return get_datapoint_value(action_context.execution_context, value[1:])
-
+            action_context.execution_context.client.connect()
```

Comment:
What's the reason for this change? Calling get_datapoint_value would save duplicate code.